fs/dafsclient/rmtclient.cpp

```diff
@@ -767,7 +770,8 @@ void CRemoteBase::connectSocket(SocketEndpoint &ep, unsigned connectTimeoutMs, u
                     }
                     else
                         ssock.setown(createSecureSocket(socket.getClear(), nullptr));
-                    int status = ssock->secure_connect();
+                    unsigned remainingMs = timer.remainingMs(connecttimeoutMs);
+                    int status = ssock->secure_connect(remainingMs);
```

Comment:
see comments above re. using same timer construct (either the CTimeMon or new CCycleTimer)